pyproject.toml

```
@@ -1,23 +1,80 @@
 [build-system]
-requires = [
-    "setuptools>=42",
-    "setuptools_scm[toml]>=3.4"
-]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
 
-[tool.setuptools_scm]
-write_to = "src/particle/version.py"
+[project]
+name = "particle"
+description = "Extended PDG particle data and MC identification codes"
+readme = "README.rst"
+requires-python = ">=3.7"
+authors = [
+    { name = "Eduardo Rodrigues", email = "eduardo.rodrigues@cern.ch" },
+    { name = "Henry Schreiner", email = "hschrein@cern.ch" },
```

Do you want to update your email address?

---

```suggestion
    { name = "Henry Schreiner", email = "henryfs@princeton.edu" },
```

Still valid but okay, maybe this is better.